sklearn/inspection/_plot/tests/test_boundary_decision_display.py

```diff
@@ -38,6 +38,18 @@ def fitted_clf():
     return LogisticRegression().fit(X, y)
 
 
+def test_input_data_dimension():
+    """Check that we raise an error when `X` does not have exactly 2 features."""
+    n_features, n_samples = 4, 10
+    X = np.random.randn(n_samples, n_features)
+    y = np.random.randint(0, 2, n_samples)
```

Comment:
Nit: Can we use `make_classification` here instead and set the `random_state` for reproducibility?

Reply:
Sure! Thanks for the suggestion, it's done :)